vtkCleaverImageToUnstructuredGridFilter.cxx

```diff
@@ -541,6 +541,13 @@ int vtkCleaverImageToUnstructuredGridFilter::RequestData(vtkInformation* vtkNotU
   // mesh->writeInfo(output_path + output_name, verbose);
   fillUnstructuredGrid(output, mesh);
 
+  // clean up memory
+  for(auto* field: fields)
```

Comment:
The field pointers in this vector are new'ed up manually in both `segmentationToIndicatorFunctions` and `loadIndicatorFunctions`--why can't these functions return vectors of unique_ptrs themselves? That's your code, not the cleaver API.